src/test/java/io/weaviate/integration/client/auth/AuthAzureClientCredentialsTest.java

```diff
@@ -1,37 +1,34 @@
 package io.weaviate.integration.client.auth;
 
-import java.io.File;
 import java.util.Arrays;
+
+import io.weaviate.integration.client.WeaviateWithAzureContainer;
 import org.apache.commons.lang3.StringUtils;
 import org.junit.Before;
 import org.junit.ClassRule;
 import org.junit.Test;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 import static org.junit.jupiter.api.Assertions.assertNotNull;
 import static org.junit.jupiter.api.Assertions.assertNull;
-import org.testcontainers.containers.DockerComposeContainer;
-import org.testcontainers.containers.wait.strategy.Wait;
 import io.weaviate.client.Config;
 import io.weaviate.client.WeaviateClient;
 import io.weaviate.client.base.Result;
 import io.weaviate.client.v1.auth.ClientCredentialsFlow;
 import io.weaviate.client.v1.auth.exception.AuthException;
 import io.weaviate.client.v1.misc.model.Meta;
+import org.testcontainers.weaviate.WeaviateContainer;
+
 import static io.weaviate.integration.client.WeaviateVersion.EXPECTED_WEAVIATE_VERSION;
 
 public class AuthAzureClientCredentialsTest {
   private String address;
 
   @ClassRule
-  public static DockerComposeContainer compose = new DockerComposeContainer(
-    new File("src/test/resources/docker-compose-azure.yaml")
-  ).withExposedService("weaviate-auth-azure_1", 8081, Wait.forHttp("/v1/.well-known/openid-configuration").forStatusCode(200));
+  public static WeaviateContainer weaviate = new WeaviateWithAzureContainer("semitechnologies/weaviate:1.23.1");
```

Comment:
would be great to set Weaviate docker version in one place. We use `EXPECTED_WEAVIATE_VERSION` to check a current version, we can use this to pass a docker image:
```
String.format("semitechnologies/weaviate:%s", EXPECTED_WEAVIATE_VERSION)
```

Reply:
done